linux/ibus-keyman/build.sh

```diff
@@ -19,11 +19,16 @@ builder_describe \
   "test" \
   "install                   install artifacts" \
   "uninstall                 uninstall artifacts" \
+  "report                    create coverage report" \
```

Comment:
Rather than making a new action for `report`, could it be an option `--report` inside the `test` action?
(The coverage report depends on test being run, right?)

Reply:
Done. Seems a bit weird to call `ninja` in response to an option, but it works.